lib/migration-payloads/validation/errors.js

```diff
@@ -50,6 +50,10 @@ const errors = {
       INVALID_VALIDATION_PROPERTY: (propName) => {
         return `A field can't have "${propName}" as a validation.`;
       },
+      INVALID_VALIDATION_PROPERTY_COMBINATION: (keys) => {
+        const formattedCombination = keys.map((key) => `"${key}"`).join(' & ');
+        return `A field can't have the combination ${formattedCombination} as a validation.`;
```

Comment:
I'd phrase this more as "x is not a valid property for a validation" The combination thing is a bit confusing.